keel-clouddriver/src/test/kotlin/com/netflix/spinnaker/keel/clouddriver/MemoryCloudDriverCacheTest.kt

```diff
@@ -69,35 +69,24 @@ object MemoryCloudDriverCacheTest {
   @Test
   fun `security groups are looked up from CloudDriver when accessed by name`() {
     every {
-      cloudDriver.getSecurityGroupSummaries("prod", "aws", "us-east-1")
-    } returns securityGroupSummaries
+      cloudDriver.getSecurityGroupSummaryByName("prod", "aws", "us-east-1", "foo")
+    } returns sg1
     every {
       cloudDriver.getCredential("prod")
     } returns Credential("prod", "aws")
 
-    subject.securityGroupByName("prod", "us-east-1", "bar").let { securityGroupSummary ->
+    subject.securityGroupByName("prod", "us-east-1", "foo").let { securityGroupSummary ->
       expectThat(securityGroupSummary) {
-        get { name }.isEqualTo("bar")
-        get { id }.isEqualTo("sg-2")
+        get { name }.isEqualTo("foo")
+        get { id }.isEqualTo("sg-1")
       }
     }
   }
 
-  @Test
-  fun `an invalid security group id throws an exception`() {
```

Comment:
Removed because it ended up being the same test as the 404 test below after refactoring for the new method names/behavior.